Runners/ConsoleApp/Program.cs

```diff
@@ -23,11 +24,14 @@ static void Main()
         void Main(IMessageSender messageSender)
         {
             var pluginExecutor = new PluginExecutor(messageSender, new MemoryDal(), new PluginsMenu(), new PluginsManager());
-
+            Console.WriteLine("Hi! I am Your amazing bot and I will be happy to be of service to you on several topics.");
+            Console.WriteLine("Please type the number of the plugin you would like to help, or type help to get the list of options.\r\nAt any point you can type MAIN or HOME to return to the main menu.");
```

Comment:
The msg here is long with \r\n
Instead, just use another WriteLine

Reply:
ok. It seems to me that the best thing would be to chain the messages with a small delay between them to give the feeling of a real chat. I just need more to learn how to do it in C#